spec/unit/installer/analyzer_spec.rb

```diff
@@ -1687,6 +1687,32 @@ module Pod
             @analyzer.analyze
           end.message.should.include 'The subspecs of `AFNetworking` are linked to different build configurations for the `Pods-SampleProject` target. CocoaPods does not currently support subspecs across different build configurations.'
         end
+
+        it 'does not implicitly whitelist a whitelisted dependency\'s transitive dependencies' do
+          SpecHelper.create_sample_app_copy_from_fixture('SampleProject')
+          @podfile = Podfile.new do
+            platform :ios, '8.0'
+            project 'SampleProject/SampleProject'
+            target 'SampleProject' do
+              pod 'AFNetworking'
+              pod 'RxCocoa', '4.0.0', :configurations => ['Debug']
+            end
+          end
+          @analyzer = Installer::Analyzer.new(config.sandbox, @podfile)
+          target_definition = @podfile.target_definitions['SampleProject']
+          aggregate_target = @analyzer.analyze.targets.find { |t| t.target_definition == target_definition }
+
+          aggregate_target.pod_targets_for_build_configuration('Debug').map(&:name).
```

Comment:
nit extract `aggregate_target.pod_targets_for_build_configuration('Debug')` to make the rest of lines single lines?

Reply:
something like 
```ruby
debug_pod_targrets.map(&:name).sort.should == [
 'PodA',
 'PodB'
]
```

Reply:
I'd suspect `AFNetworking` is also meant to be for release here right?

Reply:
yup can do